external/llvm-project/mlir/tools/mlir-opt/mlir-opt.cpp

```diff
@@ -106,6 +106,7 @@ void registerTestPreparationPassWithAllowedMemrefResults();
 void registerTestRecursiveTypesPass();
 void registerTestSCFUtilsPass();
 void registerTestVectorConversions();
+void registerTosaPartitionPass();
```

Comment:
It doesn't have to be in with the test options, but what I was copying from did that so I did too.